staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -395,6 +401,17 @@ const (
 	MigrationAbortInProgress MigrationAbortStatus = "Aborting"
 )
 
+//
+// +k8s:openapi-gen=true
+type MigrationMode string
+
+const (
+	// MigrationPreCopy means the VMI migrations that is currenly running is in pre copy mode
+	MigrationPreCopy MigrationMode = "pre-copy"
```

Comment:
pre-copy/PreCopy and post-copy/PostCopy ? woudl that be more consistent with out other values work in our api?